DoublyLinkedList/CMakeCache.txt

```diff
@@ -0,0 +1,358 @@
+# This is the CMakeCache file.
+# For build in directory: /Users/jr/Documents/Zoho-project/DoublyLinkedList
+# It was generated by CMake: /Applications/CMake.app/Contents/bin/cmake
```

Comment:
this is also a autogenned file, add to gitignore, remove this file from repo

Reply:
ok sir